website/templates/public/resetpassword.mako

```diff
@@ -1,7 +1,7 @@
 <%inherit file="base.mako"/>
 <%def name="title()">Reset Password</%def>
 <%def name="content()">
-<h1 class="page-header text-center">Reset Password</h1>
+<h1 class="page-header text-center">Reset password</h1>
```

Comment:
Top-level page titles should use title case instead of sentence case, so "Reset Password" was correct. The button text should be sentence case, "Reset password". Also, I think JIRA ticket https://openscience.atlassian.net/browse/OSF-4773 is actually referring to the forgot password page.